render/host.go

```diff
@@ -67,8 +70,21 @@ type ndmRenderer struct{}
 //Render renders disk nodes.
 func (v ndmRenderer) Render(rpt report.Report) Nodes {
```

Comment:
@Pradeepkumarbk Can we have a brief comment about what `Render` actually does?

Reply:
@satyamz addressed